kernel_tuner/core.py

```diff
@@ -337,17 +337,30 @@ def __init__(
         self.units = dev.units
         self.name = dev.name
         self.max_threads = dev.max_threads
+        self.flush_possible = lang.upper() not in ['OPENCL', 'C', 'FORTRAN'] and isinstance(self.dev.cache_size_L2, int) and self.dev.cache_size_L2 > 0
+        if self.flush_possible:
+            t = np.int32
+            self.flush_array = np.zeros((self.dev.cache_size_L2 // t(0).itemsize), order='F').astype(t)
         if not quiet:
             print("Using: " + self.dev.name)
 
-    def benchmark_default(self, func, gpu_args, threads, grid, result):
-        """Benchmark one kernel execution at a time"""
+    def flush_cache(self):
+        """This special function can be called to flush the L2 cache."""
```

Comment:
I would suggest changing the comment to:
```suggestion
        """Flush the L2 cache by overwriting it with zeros."""
```

I am surprised that this works at all, I thought that `memset` just touched the device memory.